experimental/ag-ui/front-end/src/index.css

```
@@ -1,15 +1,190 @@
+
+  /* ── Status Colors ── */
+  --status-connected: #10B981;
+  --status-connected-glow: rgba(16, 185, 129, 0.5);
+  --status-checking: #F59E0B;
+  --status-checking-glow: rgba(245, 158, 11, 0.5);
+  --status-error: #EF4444;
+  --status-error-glow: rgba(239, 68, 68, 0.5);
+  --status-disconnected: #6E7681;
+
+  /* ── Typography ── */
+  --font-primary: 'Inter', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, sans-serif;
```

Stylelint (17.7.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)